Example/Tests/UIColorPelotonTests.swift

```diff
@@ -59,6 +59,31 @@ final class UIColorPelotonSpec: QuickSpec {
       expect(view) == snapshot()
     }
 
+    it("pelotonHRZ1Blue looks right") {
+      view.backgroundColor = .pelotonHRZ1Blue
```

Comment:
I'm game for `.heartRateZone1`, but I was trying to match our design style guide (cc @sjy91523). Thoughts here Jocelyn?